supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -366,6 +366,39 @@ NOTE: When referring generally to other Kubernetes components, such as pods, nod
 
 *Incorrect forms*: Kubernetes operator, operatorize
 
+*See also*:
+[discrete]
+[[operator-framework]]
+==== image:images/yes.png[yes] Operator Framework (noun)
+*Description*: In Red Hat OpenShift, _Operator Framework_  is a family of tools and capabilities to deliver on the customer experience. It includes open source tools such as , Operator SDK, Operator Lifecycle Manager (OLM), Operator Registry, and OperatorHub.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+
+[discrete]
+[[operator-hub]]
+==== image:images/yes.png[yes] OperatorHub (noun)
```

Comment:
I'm pretty sure that alphabetically this would come after "Operator Lifecycle Manager" and before "OpEx".